src/Attributes/Param.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Consolidation\AnnotatedCommand\Attributes;
+
+use Attribute;
+use Consolidation\AnnotatedCommand\Parser\CommandInfo;
+
+#[Attribute(Attribute::TARGET_METHOD | Attribute::IS_REPEATABLE)]
+class Param implements AttributeInterface
+{
+    /**
+     * @param $name
+     *   The name of the parameter.
+     * @param $description
+     *   A one line description.
+     */
+    public function __construct(
+        public string $name,
+        public string $description
+    ) {
+    }
+
+    public static function handle(\ReflectionAttribute $attribute, CommandInfo $commandInfo)
+    {
+        $args = $attribute->getArguments();
+        $commandInfo->addArgumentDescription($args['name'], @$args['description']);
```

Comment:
The old code also had `$description = trim(preg_replace('#[ \t\n\r]+#', ' ', $param['description']));`; is that needed here, or did it move somewhere else and I just missed it?

Reply:
The help for `description` specifically says its a one line text. The `synopsis` param takes longer texts. So maybe the preg is not needed?

Reply:
Hm I am unsure whether multi-line descriptions are used anywhere, but I would tend to believe the implementation over the documentation. If we want to remove support for multi-line documentation, I'd recommend we do that in a major version bump.

Reply:
I dont think Semver applies, as we are inventing new Attribute classes. They only get used when a command author chooses to use them. The author should break description into a description and synopsis as the new class requires (matches Symfony Console help). I'm not opposed to adding the preg_replace(), I just dont think we need to.

Reply:
Ah, that's right, in the context of attributes the description is only one line. That is, in fact, better for backwards compatibility, as older PHPs won't fail to parse single-line attributes. You are right that we should require that, so the preg_repalce is not needed.